src/Language/PureScript/Publish/Utils.hs

```diff
@@ -16,14 +16,20 @@ import qualified System.FilePath.Glob as Glob
 globRelative :: Glob.Pattern -> IO [FilePath]
 globRelative pat = do
   currentDir <- getCurrentDirectory
-  filesAbsolute <- Glob.globDir1 pat currentDir
+  glob pat currentDir
+
+-- | Glob relative to the specified directory, and produce relative pathnames.

```

Comment:
However it does seem a bit odd that `glob` is using `getCurrentDirectory`; shouldn't it be using `target` instead of the result of `getCurrentDirectory`?